.circleci/config.yml

```diff
@@ -31,7 +31,7 @@ workflows:
 
       - docker-image-build:
           name: docker-image-build-pr
-          saveToWorkspace: false
+          saveToWorkspace: true  # Changed to true to test out integration tests job
```

Comment:
Since this is always true now, I think we can just remove the option to disable it, in the `docker-image-build` section of this file.